documentation/gettingstarted/Glossary.rst

```diff
@@ -1,34 +1,36 @@
+This is a list of jargon used in *GlobaLeaks* software, defined in a way so as to be unambiguous and uniform across all references.
 
 .. glossary::
-   Admin
-        The user that has setup and is maintaining the Platform.
-
-   Context
-        A topic covered by a GlobaLeaks based project (ex. Corruption).
-
    GlobaLeaks
-	Our project, a FLOSS, security focused and flexible whistleblowing framework.
-
-   Notification
-        Method used to notify a Recipient that a new Submission or an update of an existing Submission is available.
-
+	Our project, a copylefted libre software, security focused and flexible *whistleblowing* framework.
+	
    Platform
-	A system running the GlobaLeaks framework.
+	A system running the *GlobaLeaks* framework.
+	
+   Context
+        A topic covered by a *GlobaLeaks* based project (ex. corruption).
+
+   Whistleblower
+	A user that is reporting to the *platform* by uploading material and filling out the *submission* form.'
 
    Questionnaire
-        A questionnaire is a set of questions that the whistleblower should fill out for completing a Submission to a Platform.
+        A questionnaire is a set of questions that the whistleblower should fill out for completing a *submission* to a *platform*.
 
-   Receipt
-        An access token that allows the Whistleblower to be authenticated to the secret (it effectively acts as a password) generated by the Platform and returned to the Whistleblower when a Submission has been completed. It enables the Whistleblower to modify and update the submission eventually uploading new files, adding comments to the Tip, and reading Receivers's comments about the submitted materials.
+   Submission
+        The action performed by the *whistleblower* to submit a *tip*, to the *platform*, within its *context*.
+This action is performed by the *whisleblower*.
 
    Recipient
-        It is the receiver of an anonymous Submission sent by the Whistleblower.
+        The one who receives an anonymous *submission* sent by the *whistleblower*.
 
-   Submission
-        The action performed by the Whistleblower to submit a Tip to the Platform. This action is performed by the Whisleblower in respect to the Context requirement.
+   Receipt
+        An access token (effectively a password) generated by the *platform* and sent to the *whistleblower* to authenticate upon further visits to the *platform* after having completed a *submission*.
+With it, the *whistleblower* can modify and update their *submission* by uploading new files, adding comments to the *tip*, and read *recipients*'s comments about the submitted material.
 
-   Whistleblower
-	A user that is reporting to the Platform by uploading material and filling out the submission form.
+   Notification
+        A method used to notify a *recipient* of a new *submission*, or an updated existing *submission*.
+	
+   Admin
+        The user that has set up and is maintaining the *platform*.
```

Comment:
Thank you @comradekingu !

I noticed that with your review you have changed the order of the terms.

Was this intended? Maybe we could preserve the alphabetical order?

Thank you!